app/helpers/layout_helper.rb

```diff
@@ -98,6 +98,28 @@ def javascript(*args)
     content_for(:javascripts) { javascript_include_tag(*args) }
   end
 
+  def javascript_include_tag(*params, **kwargs)
+    # Workaround for overriding javasctipt load with webpack_asset_paths, should be removed when webpack_asset_paths is removed
+    if kwargs.is_a?(Hash) && kwargs[:source] == "webpack_asset_paths"
+      kwargs[:webpacked]
+    else
+      super(*params, **kwargs)
+    end
+  end
+
+  def webpack_asset_paths(plugin_name, extension: 'js')
```

Comment:
I was surprised to see why a new function showed up with just deprecations. Perhaps suggest where it came from?
```suggestion
  # @deprecated Previously provided by webpack-rails
  def webpack_asset_paths(plugin_name, extension: 'js')
```